src/AutoMapper/Mapper.cs

```diff
@@ -25,7 +25,7 @@ public class Mapper : IRuntimeMapper
         public static IConfigurationProvider Configuration
```

Comment:
Somewhat trivial, but I think here and for static mapper instance, we can use the concrete types, there is little value in using the interfaces.